proxies/live/apiproxy/targets/apim.xml

```diff
@@ -43,6 +43,13 @@
           <Name>FlowCallout.UserRoleService</Name>
           <Condition>(access-mode Equals "user-restricted")</Condition>
         </Step>
+        <!-- Get ODS code from UserInfo -->
+        <Step>
+          <Name>JavaScript.GetODSFromUserInfo</Name>
```

Comment:
The UserRoleService callout is conditional on access mode being user-restricted, which means these two new steps should probably be too